native/cocos/audio/graph_based/AudioBuffer.cpp

```diff
@@ -0,0 +1,81 @@
+#include "audio/graph_based/AudioBuffer.h"
+#include "LabSound/extended/AudioFileReader.h"
+#include "FileUtils.h"
+namespace cc {
+ccstd::unordered_map<ccstd::string, IntrusivePtr<AudioBuffer>> AudioBuffer::bufferMap;
+AudioBuffer* AudioBuffer::createBuffer(const ccstd::string& url) {
+    auto buffer = AudioBuffer::bufferMap.find(url);
+    if (buffer != AudioBuffer::bufferMap.end()) {
+        return buffer->second;
+    }
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
+    auto fileData = FileUtils::getInstance()->getDataFromFile(url);
+    ccstd::vector<uint8_t> rawData;
+    auto size = fileData.getSize();
+    rawData.resize(size);
+    memcpy(rawData.data(), fileData.getBytes(), size);
+    auto bus = lab::MakeBusFromMemory(rawData, false);
```

Comment:
Why needs to copy buffer? Could we use `Data fileData` directly?

Reply:
`lab::MakeBusFromMemory` requires a std::vector as a parameter